lib/motions/general-motions.coffee

```diff
@@ -140,9 +140,9 @@ class MoveLeft extends Motion
   operatesInclusively: false
 
   moveCursor: (cursor, count=1) ->
-    _.times count, ->
-      unless cursor.isAtBeginningOfLine()
-        cursor.moveLeft()
+    _.times count, =>
+      cursor.moveLeft() unless cursor.isAtBeginningOfLine() and !atom.config.get('vim-mode.wrapLeftRightMotion')
```

Comment:
No worries! And the `not` thing is just our preference in the Atom codebases; not something I'd expect anyone to just *know* as contributor. Thanks a lot for all of these contributions.